sdk/monitor/OpenTelemetry.Exporter.AzureMonitor/src/OpenTelemetry.Exporter.AzureMonitor.csproj

```
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <Description>An OpenTelemetry .NET exporter that exports to Azure Monitor</Description>
     <AssemblyTitle>AzureMonitor OpenTelemetry Exporter</AssemblyTitle>
```

Is `AzureMonitor` here intentional?

```suggestion
    <AssemblyTitle>Azure Monitor OpenTelemetry Exporter</AssemblyTitle>
```